browser/ui/brave_ui_features.cc

```diff
@@ -15,7 +15,7 @@ BASE_FEATURE(kBraveNtpSearchWidget,
 // Enables window cloaking on window creation to prevent a white flash.
 BASE_FEATURE(kBraveWorkaroundNewWindowFlash,
              "BraveWorkaroundNewWindowFlash",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
```

Comment:
Let's adjust `BackgroundColorTabHelperBrowserTest` to not enable the feature explicitly.